cmd/prometheus/main.go

```diff
@@ -185,6 +186,18 @@ func (c *flagConfig) setFeatureListOptions(logger log.Logger) error {
 			case "agent":
 				agentMode = true
 				level.Info(logger).Log("msg", "Experimental agent mode enabled.")
+			case "timezones":
+				if runtime.GOOS == "windows" {
+					zoneinfo := os.Getenv("ZONEINFO")
+					if len(zoneinfo) == 0 {
+						return errors.New("timezones support on Windows requires env var ZONEINFO to be supplied")
```

Comment:
We should not take configuration from environment variable, and custom timezone support could be offered for other os'es as well.